### DISCUSSION TITLE
v0.9.0 - Realtime Stable Diffusion

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#45] [General]
## Realtime Paint To Image
With the recent release of `Latent Consistency Models`(LCM) it is now possible to do things that we never thought possible even months ago.

Today we release a new version of our UI which supports live painting using LCM and any other model, this can make creating new artwork, or editing exiting artworks a lot easier and user friendly.

https://user-images.githubusercontent.com/4353684/285347896-8da6709b-fea6-4cd4-ba65-ec692401f475.mp4
 
Frame rate depends on hardware and model, please don't expect miracles, (3fps on a 3090), (1fps on 2080) is all the stats we have right now, we hope to improve this significantly in the next few releases

## Realtime Text, Image and Inpaint
Realtime is also available for TextToImage, ImageToImage and ImageInpaint pipelines

### Text To Image
https://user-images.githubusercontent.com/4353684/285347887-99db7f37-cff4-48b6-805b-3ca55e8f0c3a.mp4

### Image To Image 
https://user-images.githubusercontent.com/4353684/285348410-c19a2111-6745-4f01-8400-d137d40180fe.mp4

### Image Inpaint 
https://user-images.githubusercontent.com/4353684/285347894-9d044d7d-7c22-4379-8187-9cf7b9cac89c.mp4

## Nuget Packages Updated
https://www.nuget.org/packages/OnnxStack.Core/0.9.0
https://www.nuget.org/packages/OnnxStack.StableDiffusion/0.9.0

### What's Changed
* Core API Improvements  by @saddam213 in https://github.com/saddam213/OnnxStack/pull/33
* LCM Inpaint Diffuser by @saddam213 in https://github.com/saddam213/OnnxStack/pull/22
* Removing hash as it is different between tests despite the image being the same. by @james-s-tayler in https://github.com/saddam213/OnnxStack/pull/36
* Corrected the logger classes in the ImageInpaint and ImageToImage views by @TheyCallMeHex in https://github.com/saddam213/OnnxStack/pull/37
* Fixed timestamp being set incorrectly in ExecuteStableDiffusion in th… by @TheyCallMeHex in https://github.com/saddam213/OnnxStack/pull/38
* Realtime Stable-Diffusion by @saddam213 in https://github.com/saddam213/OnnxStack/pull/39

**Full Changelog**: https://github.com/saddam213/OnnxStack/compare/v0.8.0...v0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/saddam213/OnnxStack/releases/tag/v0.9.0'>v0.9.0 - Realtime Stable Diffusion</a>.</em>

**Comment by TheyCallMeHex** [comment]
We have a Youtube channel here: https://www.youtube.com/@OnnxStack

We also have a Discord server here: https://discord.gg/D3gEYWru5h

Feel free to post images etc in the artwork channel on Discord, we'd love to see what people make using OnnxStack.

**Comment by saddam213** [comment]
https://github.com/saddam213/OnnxStack/assets/4353684/212ad4e6-3735-4fb0-b01f-495a182585c9


https://user-images.githubusercontent.com/4353684/285547207-3a7ea067-fcbf-47f0-9372-fafa94d301f7.mp4

**Comment by Amin456789** [comment]
THANKSSS! Been waiting for this update!
